remotespark/livyclientlib/exceptions.py

```diff
@@ -32,3 +46,53 @@ class SessionManagementException(LivyClientLibException):
 class BadUserDataException(LivyClientLibException):
     """An exception that is thrown when data provided by the user is invalid
     in some way"""
+
+
+# == DECORATORS FOR EXCEPTION HANDLING ==
+def handle_expected_exceptions(f):
+    """A decorator that handles expected exceptions. Self can be any object with
+    an "ipython_display" attribute.
+    Usage:
+    @handle_expected_exceptions
+    def fn(self, ...):
+        etc..."""
+    exceptions_to_handle = (BadUserDataException, LivyUnexpectedStatusException, FailedToCreateSqlContextException,
```

Comment:
I meant we should put this list at the top of the file, like a constant or something so that it's easily discoverable.